events.html

```diff
@@ -0,0 +1,141 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Google Calendar API Quickstart</title>
+    <meta charset="utf-8" />
+  </head>
+  <body>
+    <p>Google Calendar API Quickstart</p>
+
+    <!--Add buttons to initiate auth sequence and sign out-->
+    <button id="authorize_button" style="display: none;">Authorize</button>
+    <button id="signout_button" style="display: none;">Sign Out</button>
+
+    <pre id="content" style="white-space: pre-wrap;"></pre>
+
+    <script type="text/javascript">
+      // Client ID and API key from the Developer Console
+      var CLIENT_ID = '<569267638609-qmrioa5fa1qhbe4b2tmnllbtrv2ltilm.apps.googleusercontent.com>';
+      var API_KEY = '<AIzaSyCdTnv7lgEhQSm2jJvo-aZaL3vBni1hb-A>';
```

Comment:
Question: Did you restrict the API key to a specific domain when running the application? From what I've researched, it seems like it's ok to embed the API_KEY here, but it should be restricted to our domain so that other sites cannot use this key and pretend to be us (we may need to come up with a domain to help deploy from). Here's the documentation for reference: https://cloud.google.com/docs/authentication/api-keys#adding_http_restrictions.

Alternatively, it would also be possible to run this code from the server-side using the Java examples if you don't want to restrict the API key. That's completely up to you though :)